src/Sylius/Behat/Context/Setup/ProductContext.php

```diff
@@ -540,6 +558,33 @@ public function thisProductHasAnImageWithACode(ProductInterface $product, $image
     }
 
     /**
+     * @Given /^(it) has different prices for different channels and currencies$/
+     */
+    public function itHasDifferentPricesForDifferentChannelsAndCurrencies(ProductInterface $product)
+    {
+        $product->getFirstVariant()->setPricingCalculator(Calculators::CHANNEL_AND_CURRENCY_BASED);
```

Comment:
`getFirstVariant` method should be removed. `defaultVariantResolver` should be used instead.